static/app/components/events/eventReplay/replayClipSection.tsx

```diff
@@ -65,10 +65,13 @@ export function ReplayClipSection({event, group, replayId}: Props) {
     replayCount && replayCount > 1 ? (
       <Fragment>
         <div>
-          {t(
-            'There are %s for this issue.',
-            tn('%s replay', '%s replays', replayCount ?? 0)
-          )}
```

Comment:
This was never the right way to translate the sentence. Using `tn()` inside a `t()`... or any other combination of nesting like that... is going to make it hard for translators and lead to bad translations.